.github/workflows/rw-doc-release-apidocs.yml

```diff
@@ -56,14 +56,10 @@ jobs:
             git config --global user.email "git-action@gooddata.com"
             git config --global user.name "git-action"
 
-      - name: Setup node
-        uses: actions/setup-node@v4
-        with:
-          node-version: 18.17.0
-
-      - name: Install rush
-        run: |
-            npm install -g @microsoft/rush
+      - name: Setup rush
+        env:
+          NPM_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        uses: ./gooddata-ui-sdk/.github/actions/rush/set-up-rush
```

Comment:
Not sure about this path ? 

Hope if we do checkout with path gooddata-ui-sdk than result path to action is ./gooddata-ui-sdk/.github/actions/rush/set-up-rush but this local action using set-up-node on path started ./ ... so ?

Reply:
when I call it from master 

gooddata/gooddata-ui-sdk/.github/actions/rush/set-up-rush@master

I got this error 
 Can't find 'action.yml', 'action.yaml' or 'Dockerfile' under '/home/runner/_work/gooddata-ui-sdk/gooddata-ui-sdk/.github/actions/node/set-up-node'.